index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <title>Profolio</title>
+</head>
+<body>
+
+        <div class="container">
+
+            <section class="Header">
+                
+                    <ul class="header-logo">
+                        <li class="logo"> My Logo</li>
+                        <li class="hamburg-logo"> <img src="Img/Hlogo.png" alt=""></li>
+                
+                    </ul>
```

Comment:
Great job implementing the nav👍. Kindly make use of `header` tag and `nav` tag for this section, this is in accordance to HTML semantic tag usage.
```
`<header>
   <a href="#"> LOGO</a>
    <nav>
    <img src="Img/Hlogo.png" alt="">
   </nav>
 </header>`
```

Kindly note that your logo should be wrapped in an `a` tag so that users can navigate back to the home page by clicking on the logo. 😎